File: NextEvent/next-event-front/src/interfaces/Evento.ts
export default interface Evento{
    id?: string;
    nome: string;
    descricao: string;
    // . Precisa ver qual o tipo de dado que tem que colocar nas datas, eu não lembro, no DB é DateTime
    dataInicio: string; 
    dataFim: string;
    // . 
    criadoEm?: string;
    ativo?: boolean;
}

File: NextEvent/NextEventDB/Models/Evento.cs
namespace NextEvent.Models;

public class Evento
{
    public Evento()
    {
        CriadoEm = DateTime.Now;
        Ativo = true;
        Nome = string.Empty;
        Descricao = string.Empty;
    }

    public int Id { get; set; }
    public string Nome { get; set; }
    public string Descricao { get; set; }
    public DateTime DataInicio { get; set; }
    public DateTime DataFim { get; set; }
    public bool Ativo { get; set; }
    public DateTime CriadoEm { get; set; }
    public int AdministradorId { get; set; }

}